beacon-chain/forkchoice/doubly-linked-tree/BUILD.bazel

```diff
@@ -58,6 +58,9 @@ go_test(
         "vote_test.go",
     ],
     embed = [":go_default_library"],
+    visibility = [
+        "//beacon-chain:__subpackages__",
+    ],
```

Comment:
I don't remember the exact rationale, but we agreed a long ago that we should just use public visibility. You can ask Kasey for details as he was the one advocating it.

Reply:
this was a manual mistake, there should  be no visibility statement here.